src/main/java/org/gridsuite/odre/server/dto/LineGeoData.java

```diff
@@ -25,5 +25,7 @@
 
     private String country2;
 
+    private String substationStart;
```

Comment:
to make things more symmetric and easy, can we store substationEnd as well ?

Reply:
This would allow to query lines without knowing their ids, just by the end substations